src/integration-tests/test_admin_client.py

```diff
@@ -7,7 +7,11 @@
 import re
 from typing import Dict, Optional, Tuple
 
-from blazingmq.dev.it.fixtures import Cluster, single_node, order  # pylint: disable=unused-import
+from blazingmq.dev.it.fixtures import (
+    Cluster,
+    single_node,
+    order,
```

Comment:
`order` is not used. See example [here](https://github.com/bloomberg/blazingmq/blob/main/src/integration-tests/test_breathing.py#L22).

Also, let's re-order the imports with `isort --profile black`:

```
from blazingmq.dev.it.fixtures import (  # pylint: disable=unused-import
    Cluster,
    order,
    single_node,
)
```

Reply:
Thanks @jll63!
Set up the order, applied formatting and imports reordering